how-to-guides/sagemaker/README.md

```diff
@@ -0,0 +1,12 @@
+# Using Neptune client with AWS SageMaker
+
+The following examples show how to use Neptune client with AWS SageMaker. The examples use Jupyter notebooks. The
+notebooks in most cases should be run from AWS SageMaker notebooks or AWS SageMaker Studio notebooks, the information
+regarding how they should be used is provided within the notebooks.
+
+For running the example of using the custom docker container, you will additionally need to install [docker] and
+[AWS CLI] tools.
+
+
+ [docker]: https://docs.docker.com/engine/install/
+ [AWS CLI]: https://aws.amazon.com/cli/
```

Comment:
I meant updating the [repository readme](https://github.com/neptune-ai/examples#readme) :)

Reply:
Done.